COCOAPODS.md

```diff
@@ -1,150 +1,343 @@
+### Add a dependency on a Pod library from the CocoaPods repository
 
-        // The name of the produced framework can be changed.
-        // The name of the Gradle project is used here by default.
-        frameworkName = "my_framework"
+1. Add dependencies on a Pod library that you want to use from the CocoaPods repository with `pod()`  to `build.gradle.kts` 
+(`build.gradle`) of your project.
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("AFNetworking", "~> 3.2.1")
+        }
     }
-}
-```
+    ```
+    
+    </div>
 
-</div>
+2. Re-import the project.
 
-The following podspec fields are required by CocoaPods:
-  - `version`
-  - `summary`
-  - `homepage`
+To use this dependency from Kotlin code, import a package `cocoapods.<library-name>`.
+In the example above, it's `cocoapods.AFNetworking`.
+
+### Add a dependency on an Objective-C Pod library stored locally
+
+1. Add a dependency on a Pod library stored locally with `pod()` to `build.gradle.kts` (`build.gradle`) of your
+ project.  
+As the third argument, specify the path to `Podspec` of the local Pod using `projectDir.resolve("..")`.  
+    > You can also add dependencies as subspecs.  
+    > The `cocoapods` block can include dependencies to Pods stored locally and Pods from the CocoaPods repository at
+    > the same time.
+    {:.note}
 
-A version of the Gradle project is used as a value for the `version` field.
-Fields`summary` and `homepage` can be configured using the `cocoapods` code block.
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("pod_dependency", "1.0", projectDir.resolve("../pod_dependency/pod_dependency.podspec"))
+            pod("subspec_dependency/Core", "1.0", projectDir.resolve("../subspec_dependency/subspec_dependency.podspec"))
+            pod("AFNetworking", "~> 3.2.1")
+        }
+    }
+    ```
+    
+    </div>
 
-This podspec file can be referenced from a [Podfile](https://guides.cocoapods.org/using/the-podfile.html)
-of an Xcode project. After that the framework built from the Kotlin/Native module can be used from
-this Xcode project. If necessary, this framework is automatically rebuilt during Xcode build process.
+2. Re-import the project.
 
-## Workflow
+You can find a sample project [here](https://github.com/zoldater/KotlinWithLocalObjCPods). 
+ 
+## Use a Kotlin Gradle project as a CocoaPods dependency
 
-To import a Kotlin/Native module in an existing Xcode project:
+You can use a Kotlin/Native Gradle project as a CocoaPods dependency (Kotlin Pod). Such a representation provides the
```

Comment:
More precisely, Kotlin Pod is Multiplatform Project with Native targets. It also requires multiplatform plugin to be applied.

Reply:
Thx. Will add.

Reply:
Updated a description of a Kotlin Pod. There is information about the multiplatform plugin in the section about _Installing plugins._